device/arista/x86_64-arista_7800_sup/Arista-7804R3-FM/10/port_config.ini

```diff
@@ -0,0 +1 @@
+# name         lanes                     alias        index  speed
```

Comment:
Do we need this empty port_config.ini files for fabric modules?

Reply:
It used to be mandatory to prevent processes from crashing when we first brought up the fabrics.
Today it might not be mandatory anymore but it would take some time to verify.
To prevent further delays in merging this PR, I believe it's fine to keep the empty files as is for now as they do not cause any harm while removing them might.